cobalt/browser/client_hint_headers.cc

```diff
@@ -43,12 +43,17 @@ std::vector<std::string> GetClientHintHeaders(
     const UserAgentPlatformInfo& platform_info) {
   std::vector<std::string> headers;
 
+  AddHeader(headers, "Cert-Scope", platform_info.cert_scope());
```

Comment:
Just a comment, no need to change, but i'd narrow the naming to be more specific to Youtube Livingroom. Maybe `YTLR-Cert-Scope` or `Youtube-Certification-Scope` ? The concept is only defined in LR cert requirements though.

I'd expect this to provide more context for anyone looking at server logs or data pipelines.